app/src/main/java/com/github/h3lp3rs/h3lp/forum/ForumPostsActivity.kt

```diff
@@ -0,0 +1,69 @@
+package com.github.h3lp3rs.h3lp.forum
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.github.h3lp3rs.h3lp.R
+import com.github.h3lp3rs.h3lp.forum.data.ForumPostData
+import com.github.h3lp3rs.h3lp.storage.Storages
+import com.github.h3lp3rs.h3lp.storage.Storages.Companion.storageOf
+import com.xwray.groupie.GroupAdapter
+import com.xwray.groupie.ViewHolder
+import kotlinx.android.synthetic.main.activity_forum_answers.*
+import kotlinx.android.synthetic.main.activity_forum_posts.*
+
+/**
+ * Activity containing the forum posts of a given a category
+ */
+class ForumPostsActivity : AppCompatActivity() {
+    companion object {
+        // The post of which the user wants to see the answers
+        lateinit var selectedPost : ForumPost
```

Comment:
Not sure about this like I already said, but if it is too complicated to do otherwise, whatever just keep it.